src/ontology/cl-edit.owl

```diff
@@ -23174,6 +23174,17 @@ AnnotationAssertion(rdfs:label obo:CL_0017001 "splanchnic mesodermal cell")
 SubClassOf(obo:CL_0017001 obo:CL_0011010)
 SubClassOf(obo:CL_0017001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004872))
 
+# Class: obo:CL_0017002 (prostate neuroendocrine cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33842835"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://orcid.org/0000-0001-7772-9578"^^xsd:string) obo:IAO_0000115 obo:CL_0017002 "A neuroendocrine cell that is part of the prostate epithelium.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0017002 <http://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0017002 "neuroendocrine cell of prostate epithelium")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0017002 "neuroendocrine cell of the prostate")
+AnnotationAssertion(rdfs:label obo:CL_0017002 "prostate neuroendocrine cell")
+EquivalentClasses(obo:CL_0017002 ObjectIntersectionOf(obo:CL_0000165 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000428)))
+SubClassOf(obo:CL_0017002 obo:CL_0000165)
+SubClassOf(obo:CL_0017002 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000428))
```

Comment:
No need to duplicate EquivalentTo axiom clauses as SubClassOf axioms

Reply:
@dosumis is anything still left here to do?
 I am sorry for the long delay by the way.